MyOnboarding/test/MyOnboardingApp.Api.Tests/Controllers/TodoListControllerTests.cs

```diff
@@ -42,10 +50,22 @@ public async Task Get_NoIdSpecified_ReturnsCorrectResponse()
         {
             var expectedItems = new[]
             {
-                new TodoListItem {Text = "1st Todo Item", Id = new Guid("11111111-1111-1111-1111-aabbccddeeff")},
-                new TodoListItem {Text = "2nd Todo Item", Id = new Guid("22222222-2222-2222-2222-aabbccddeeff")}
+                new TodoListItem
+                {
+                    Text = "1st Todo Item",
+                    Id = new Guid("11111111-1111-1111-1111-aabbccddeeff"),
+                    CreationTime = new DateTime(1995, 01, 01),
+                    LastUpdateTime = new DateTime(1996, 01, 01)
+                },
+                new TodoListItem
+                {
+                    Text = "2nd Todo Item",
+                    Id = new Guid("22222222-2222-2222-2222-aabbccddeeff"),
+                    CreationTime = new DateTime(2000, 01, 01),
+                    LastUpdateTime = new DateTime(2001, 01, 01)
+                }
             };
-            _repository.GetAllItemsAsync().Returns(expectedItems);
+            _obtainService.GetAllItemsAsync().Returns(expectedItems);
 
             var message = await _controller.GetMessageFromActionAsync(controller => controller.GetAsync());
             message.TryGetContentValue(out TodoListItem[] itemsFromMessage);
```

Comment:
You are too specific here, let the production code decide what collection will it return (it does not matter to the response anyway (as far as it is `IEnumerable`)):
```suggestion
            message.TryGetContentValue(out IEnumerable<TodoListItem> itemsFromMessage);
```